lib/system/external.toit

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2024 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
+
+import io
+import rpc
+
+class ExternalMessageHandler_ implements SystemMessageHandler_:
+  static TYPE ::= SYSTEM-EXTERNAL-NOTIFICATION_
+  handler/Lambda := ?
+
+  constructor .handler:
+    set-system-message-handler_ TYPE this
+
+  close -> none:
+    clear-system-message-handler_ TYPE
+
+  on-message type/int gid/int pid/int argument -> none:
+    handler.call argument
+
+externals_ := {:}
+
+class External:
```

Comment:
So this will naturally be called `external.External`? It's a bit weird.

Client, Target, Reference ...